### ISSUE TITLE
Modifier Mouse-Buttons (ctrl+Right-Click) Keybinds don't work.

### ISSUE THREAD

**Issue by ZINK-ZINK** [Snakybo/Clicked#213]
### Describe the bug

There's no Error to report.

Clicked works as expected with just left/right Mouse-Buttons, but when trying to execute ctrl+click it seems to ignore the Clicked bindings and just Target or open the Menu.

I tried resetting the default Blizzard Click-Bindings, and I triple-checked that Target & Menu are bound to SHIFT+click.

Testing with Pitbull 4 Unit Frames, and as Clicked macros, with no extra options selected. 

### Clicked Version

1.15.5

### World of Warcraft Flavor

Retail

### Tried with only Clicked enabled

- [ ] Yes
- [X] No

### Reproduction Steps

.

**Comment by ZINK-ZINK** [comment]
Nevermind, I found the problem.

I didn't change anything in the "Target" options, and I created each Binding the exact same way, but for some reason all the modifier+click Bindings defaulted to a different option and that made them not work.

I changed all the modifier Bindings to match the working ones and everything seems to work properly now.

So my issue is fixed, BUT as the Developer you might want to change how that default behavior works, because it's confusing and non-obvious to New Users.
